listenbrainz_spark/stats/sitewide/entity.py

```diff
@@ -25,105 +23,70 @@
     'artists': SitewideArtistRecord
 }
 
-time_range_schema = ["time_range", "from_ts", "to_ts"]
-
 
 def get_entity_week(entity: str) -> Optional[List[SitewideEntityStatMessage]]:
     """ Get the weekly sitewide top entity """
     logger.debug(f"Calculating sitewide_{entity}_week...")
 
     to_date = get_last_monday(get_latest_listen_ts())
-    # Set time to 00:00
-    to_date = datetime(to_date.year, to_date.month, to_date.day)
-    from_date = offset_days(to_date, 14)
-    day = from_date
-
-    # Genarate a dataframe containing days of last and current week along with start and end time
-    time_range = []
-    while day < to_date:
-        time_range.append([day.strftime('%A %d %B %Y'), int(day.timestamp()), int(get_day_end(day).timestamp())])
-        day = offset_days(day, 1, shift_backwards=False)
-
-    time_range_df = listenbrainz_spark.session.createDataFrame(time_range, schema=time_range_schema)
-    time_range_df.createOrReplaceTempView('time_range')
+    from_date = offset_days(to_date, 7)
```

Comment:
Previously the range was 14 days, now is 7 days. Why has this changed?

Reply:
This used to calculate the top artitst stats for each day of the last two weeks earlier. Not sure why because for user stats, we just calculate the overall top artist of the current week. So, I changed 14 to 7 to make it consistent.

Reply:
:+1: